### PR TITLE
[2530] Solve duplicate user error by renaming old users email to an example one

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

https://trello.com/c/Na8GbbZ1/2530-issues-for-authenticationserviceduplicateusererror

### Changes proposed in this pull request

* We change the duplicate users email to an example one so we no longer have duplicate user issue

### Guidance to review

* Create duplicate user with same email but without dfe id
* Try to login

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Comment by mooktakim** [comment]
[Issues for AuthenticationService::DuplicateUserError](https://trello.com/c/Na8GbbZ1/2530-issues-for-authenticationserviceduplicateusererror)

File: spec/services/authentication_service_spec.rb [hunk 1]
```diff
@@ -71,16 +71,12 @@
 
         it { is_expected.to eq user }
 
-        it "does not update the user's email" do
-          expect { subject }.not_to(change { user.reload.email })
+        it "update's the user's email" do
```
Comment (gpeng) [inline spec/services/authentication_service_spec.rb:74]:
```suggestion
        it "updates the user's email" do
```

File: spec/services/authentication_service_spec.rb [hunk 2]
```diff
@@ -71,16 +71,12 @@
...
+          expect { subject }.to(change { user.reload.email }.to(email))
         end
 
-        it "generates an exception which is captured by Sentry" do
-          expect(Sentry).to receive(:capture_exception).with(
-            instance_of(AuthenticationService::DuplicateUserError),
-          )
-
-          subject
+        it "existing user update's the email to example email" do
```
Comment (gpeng) [inline spec/services/authentication_service_spec.rb:78]:
```suggestion
        it "updates the existing users email to example email" do
```

**Review by gpeng — APPROVED** [review]
Couple of minor comments with the spec descriptions but looks good otherwise 👍

**Review by gpeng — APPROVED** [review]

**Pull request merged**